keyboards/bastardkb/charybdis/readme.md

```diff
@@ -18,32 +18,61 @@ Check out the [keyboard build guides](https://docs.bastardkb.com) for the Charyb
 
 ## Building the firmware
 
+You must specify the shield version when compiling/flashing the firmware. 
+
 ### Charybdis (4x6)
 
 The `default` keymap is inspired from the original [Dactyl Manuform](../../handwired/dactyl_manuform) layout.
 
+* For Elite-C v1 shield
+
 ```shell
-qmk compile -kb bastardkb/charybdis/4x6 -km default
+qmk compile -kb bastardkb/charybdis/4x6/1.x/elitec -km default
```

Comment:
This changed recently after updating the QMK's `master` branch.  The dot (`.`) is no longer a valid character in the keyboard path.  It should be:

- `bastardkb/charybdis/3x5/v1/elitec`
- `bastardkb/charybdis/3x5/v2/elitec`
- `bastardkb/charybdis/4x6/v1/elitec`
- `bastardkb/charybdis/4x6/v2/elitec`

Reply:
updated